tests/eip7702/test_eip7702_tuple_userop.py

```diff
@@ -0,0 +1,126 @@
+import pytest
+from tests.transaction_eip_7702 import TupleEIP7702
+from tests.types import CommandLineArgs, RPCErrorCode
+from tests.utils import (
+    UserOperation,
+    assert_ok,
+    assert_rpc_error,
+    deploy_contract,
+    userop_hash,
+    send_bundle_now,
+)
+
+AUTHORIZED_ACCOUNT_PREFIX = "ef0100"
+
+last_account = None
+
+
+@pytest.fixture
+def impl7702(w3):
+    return deploy_contract(w3, "SimpleWallet", ctrparams=[CommandLineArgs.entrypoint])
+
+
+def test_send_eip_7702_tx(w3, userop, impl7702, wallet_contract, helper_contract):
+    acc = w3.eth.account.create()
+    # fund the EOA address
+    w3.eth.send_transaction(
+        {"from": w3.eth.accounts[0], "to": acc.address, "value": 10**18}
+    )
+
+    # create an EIP-7702 authorization tuple
+    nonce = w3.eth.get_transaction_count(acc.address)
+    auth_tuple = TupleEIP7702(
+        chainId=hex(1337), address=impl7702.address, nonce=hex(nonce)
+    )
+    auth_tuple.sign(acc._private_key.hex())
+
+    userop.sender = acc.address
+    userop.authorizationList = [auth_tuple]
```

Comment:
just curious - for a single user operation, is that possible to attach multiple authorization tuples? and if so, what would be the use case? 

https://hackmd.io/WJtBdwvnT5yLq90F7UilIQ is pointing a single authorization tuples UO structure which I personally think it makes lot of sense.